### PR TITLE
4.0.9

### PR DESCRIPTION
**Pull request by DavidKimVungle**
@Goles @ashinagawa @annavungle @gilbertchoy Please review the changes

### CODE REVIEW

File: Sample/Vungle Sample App/FirstViewController.m [hunk 1]
```diff
@@ -58,22 +55,12 @@ - (void)vungleSDKwillShowAd {
...
-        //Animations / Sound / Gameplay can be resumed now
-        NSLog(@"The ad presented was not tapped - the user has returned to the app");
-        NSLog(@"ViewInfo Dictionary:");
-        for(NSString * key in [viewInfo allKeys]) {
-            NSLog(@"%@ : %@", key, [[viewInfo objectForKey:key] description]);
-        }
+- (void)vungleSDKwillCloseAdWithViewInfo:(NSDictionary *)viewInfo willPresentProductSheet:(BOOL)willPresentProductSheet {
+    NSLog(@"ViewInfo Dictionary:");
+    // viewInfo dictioanry contains completedView, playTime and didDownload information
+    // willPresentProduct always returns NO
```
Comment (annavungle) [inline Sample/Vungle Sample App/FirstViewController.m]: willPresentProductSheet ?

File: Sample/Vungle Sample App/FirstViewController.m [hunk 2]
```diff
@@ -84,14 +71,13 @@ - (void)vungleSDKwillCloseProductSheet:(id)productSheet {
 
 #pragma mark - FirstView Methods
 
-- (void) enableAdButtons:(BOOL)enabled {
+- (void)enableAdButtons:(BOOL)enabled {
```
Comment (annavungle) [inline Sample/Vungle Sample App/FirstViewController.m:74]: Awesome, thank you David!

File: Sample/Vungle Sample App/FirstViewController.m [hunk 3]
```diff
@@ -84,14 +71,13 @@ - (void)vungleSDKwillCloseProductSheet:(id)productSheet {
...
     _showAdButton.enabled = enabled;
     _showAdWithOptionsButton.enabled = enabled;
 	_showIncentivizedAdButton.enabled = enabled;
```
Comment (annavungle) [inline Sample/Vungle Sample App/FirstViewController.m]: Sorry can you match spacing of this line?

File: Sample/Vungle Sample App/FirstViewController.m (hunk 2 above)
Comment (DavidKimVungle) [reply Sample/Vungle Sample App/FirstViewController.m:74]: Yes ma'am!

**Comment by annavungle** [comment]
Looks great, thanks!

**Pull request merged**